builder/proxmox/config.go

```diff
@@ -213,6 +214,9 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 	if c.Node == "" {
 		errs = packer.MultiErrorAppend(errs, errors.New("node must be specified"))
 	}
+	if strings.ContainsAny(c.TemplateName, " ") {
+		errs = packer.MultiErrorAppend(errs, errors.New("template_name must not contain spaces"))
+	}
```

Comment:
Has there been a merge the wrong direction or something in the PR? This change was merged into master in #8799.
Also see that there's a bunch of other changes which don't seem related to the vsphere-iso builder.

Reply:
I think I rebased incorrectly. Trying again...I think it looks better now.